src/tree/SlotTreeItemBase.ts

```diff
@@ -3,215 +3,103 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { SiteConfig, SiteSourceControl, StringDictionary } from '@azure/arm-appservice';
-import { AppSettingsTreeItem, AppSettingTreeItem, deleteSite, DeploymentsTreeItem, DeploymentTreeItem, getFile, LogFilesTreeItem, ParsedSite, SiteFilesTreeItem } from '@microsoft/vscode-azext-azureappservice';
+import { AppSettingsTreeItem, DeploymentsTreeItem, ParsedSite } from '@microsoft/vscode-azext-azureappservice';
 import { AzExtParentTreeItem, AzExtTreeItem, IActionContext, TreeItemIconPath } from '@microsoft/vscode-azext-utils';
-import { runFromPackageKey } from '../constants';
-import { IParsedHostJson, parseHostJson } from '../funcConfig/host';
-import { FuncVersion, latestGAVersion, tryParseFuncVersion } from '../FuncVersion';
-import { envUtils } from '../utils/envUtils';
-import { nonNullValue } from '../utils/nonNull';
-import { treeUtils } from '../utils/treeUtils';
+import { IParsedHostJson } from '../funcConfig/host';
+import { FuncVersion } from '../FuncVersion';
 import { ApplicationSettings, FuncHostRequest, IProjectTreeItem } from './IProjectTreeItem';
-import { matchesAnyPart, ProjectResource, ProjectSource } from './projectContextValues';
-import { RemoteFunctionsTreeItem } from './remoteProject/RemoteFunctionsTreeItem';
+import { ProjectSource } from './projectContextValues';
+import { ResolvedFunctionAppResource } from './ResolvedFunctionAppResource';
 
-export abstract class SlotTreeItemBase extends AzExtParentTreeItem implements IProjectTreeItem {
+export class SlotTreeItemBase extends AzExtParentTreeItem implements IProjectTreeItem {
```

Comment:
Since this is no longer abstract shouldn't we just rename it to `SlotTreeItem`?

Reply:
Yeah, it's a little bit confusing since it is technically the production (top level) app _and_ the `SlotTreeItem`.  I'm not sure keeping it as `base` really helps distinguish that though 😅